### PR TITLE
Fix visibility of elements distributed into an iron-pages inside another component in Safari 11

### LINKED ISSUE #65: Selected page is not displayed on Safari 11 when content comes from slot
### Description
When a component contains an iron-pages, and the content of the iron-pages is a slot, the selected page injected into the iron-pages from the light dom is not rendered.

### Expected outcome
Selected page is rendered.

### Actual outcome
No page is rendered.
It seems that the :not(.iron-selected) selector captures the slot itself, changing its display to none and in so doing not displaying any of the elements that replace the slot. Changing it to :not(slot):not(.iron-selected) in the inspector seems to fix it.

### Live Demo
https://output.jsbin.com/piqocih

### Steps to reproduce
1. Create a custom element
2. Put an iron-pages in the custom element's template.
3. Set the content of the iron-pages to a slot.
4. Instantiate the custom element, passing elements that will be put in that slot.

### Browsers Affected
- [ ] Chrome
- [ ] Firefox
- [X] Safari 11
- [ ] Safari 10
- [ ] Safari 9
- [ ] Safari 8
- [ ] Safari 7
- [ ] Edge
- [ ] IE 11
- [ ] IE 10

### PR DESCRIPTION
**Pull request by Torgen**
The issue seems to be that Safari implements slot tags as having "display: content". When you put a slot inside an iron-pages, it gets "display: none !important", which causes Safari not to display the nodes that should be distributed into it. Includes test case.

Fixes #65

### CODE REVIEW

**Comment by Torgen** [comment]
CI issue is that the project doesn't have Sauce credentials in Travis. It needs them, since it only affects Safari, which Travis won't test without Sauce.

File: wct.conf.json
Comment (keanulee) [inline wct.conf.json]: Is this file part of the PR?

Comment (bicknellr) [reply wct.conf.json]: Maybe this is why the tests are failing?

Comment (Torgen) [reply wct.conf.json]: The default WCT config (at least when I created the pull request--it may have changed since) didn't include Safari 11 on Sauce yet, which was an issue because it's the only browser on which this issue exists. Therefore I had to add a wct config that forced Safari 11 to be tested. Which in turn leads to the CI failure because the Travis project doesn't have associated Sauce credentials. If WCT has updated their defaults, I can remove this.

Comment (Torgen) [reply wct.conf.json]: ...but even if WCT has updated their defaults, Travis still needs to be set up to use Sauce, or it will only run tests locally, meaning Firefox and Chrome, which won't test that my change actually fixes the underlying issue.

Comment (keanulee) [reply wct.conf.json]: FWIW, patch seems good when I tested with Safari 11 on my local machine.

**Review by e111077 — APPROVED** [review]
I think it would be good to get this through and figure out the sauce thing in a separate PR as it very much seems as if it works in local tests, this issue is not due to setup and not code, and travis has been weird lately anyway.

I'll merge if I get another approval.

**Comment by samuelli** [comment]
@Torgen @e111077: Sauce doesn't run on pull requests. This is due to encrypted variables not being accessible in untrusted builds. See https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml for more info on that.

File: wct.conf.json
Comment (samuelli) [reply wct.conf.json]: Travis here is already set up to run Sauce. https://github.com/PolymerElements/iron-pages/blob/master/.travis.yml#L23

But it doesn't run Safari 11.

**Review by samuelli — CHANGES_REQUESTED** [review]
Can you remove wct.conf.json please?

File: wct.conf.json
Comment (samuelli) [reply wct.conf.json]: I'm updating the default browser list to run Safari 11:
https://github.com/Polymer/wct-sauce/pull/29

**Comment by Torgen** [comment]
Done.

**Review by samuelli — APPROVED** [review]

**Pull request merged**
